index.html

```diff
@@ -1,324 +1,210 @@
-            <h2>Profesional Art Printing Data</h2>
-            <p>
-              A daily selection of privately personalized reads no accounts or
-              sign-ups required. has been the industry's standard
-            </p>
-            <div class="languages">
-              <ul>
-                <li class="project-language">html</li>
-                <li class="project-language">bootstrap</li>
-                <li class="project-language">Ruby</li>
-              </ul>
-            </div>
-            <button type="button" class="project-btn">See project</button>
-          </div>
-          <div class="project">
-            <h2>Profesional Art Printing Data</h2>
-            <p>
-              A daily selection of privately personalized reads no accounts or
-              sign-ups required. has been the industry's standard
-            </p>
-            <div class="languages">
-              <ul>
-                <li class="project-language">html</li>
-                <li></li>
-                <li class="project-language">bootstrap</li>
-                <li></li>
-                <li class="project-language">Ruby</li>
-                <li></li>
-              </ul>
+        <section class="container">
+            <div class="text">
+                <h1> Hey There. I'm Annah</h1>
+                <h3>I'm a Software Developer.</h3>
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
             </div>
-            <button type="button" class="project-btn">See project</button>
-          </div>
-          <div class="project">
-            <h2>Profesional Art Printing Data</h2>
-            <p>
-              A daily selection of privately personalized reads no accounts or
-              sign-ups required. has been the industry's standard
-            </p>
-            <div class="languages">
-              <ul>
-                <li class="project-language">html</li>
-                <li></li>
-                <li class="project-language">bootstrap</li>
-                <li></li>
-                <li class="project-language">Ruby</li>
-                <li></li>
-              </ul>
+            <div class="icon">
+                <ul>
+                    <li><a href="#"><img src="./images/img1.png" alt="Github-icon"></a></li>
+                    <li><a href="#"><img src="./images/img2.png" alt="LinkeIn-icon"></a></li>
+                    <li><a href="#"><img src="./images/img3.png" alt="AngelList-icon"></a></li>
+                    <li><a href="#"><img src="./images/img4.png" alt="Twitter-icon"></a></li>
+                    <li><a href="#"><img src="./images/img5.png" alt="Medium-icon"></a></li>
+                </ul>
             </div>
-            <button type="button" class="project-btn">See project</button>
```

Comment:
Please add hover and active styles to buttons by following the design.

Reply:
Thanks @SajjadAhmad14  i will work on the requested changes!